client/src/cmdhficlass.c

```diff
@@ -3842,6 +3842,304 @@ void picopass_elite_nextKey(uint8_t* key) {
     memcpy(key, key_state, 8);
 }
 
+static int CmdHFiClassRecover(uint8_t key[8]) {
+
+    uint32_t payload_size = sizeof(iclass_recover_req_t);
+    uint8_t aa2_standard_key[PICOPASS_BLOCK_SIZE] = {0xFD, 0xCB, 0x5A, 0x52, 0xEA, 0x8F, 0x30, 0x90};
```

Comment:
would be better to reuse the keys array in the top of this file for this purpose.